app/controllers/branches_controller.rb

```diff
@@ -28,15 +29,17 @@ def edit
 
 	def update
 		if @branch.update(branch_params)
-			redirect_to @branch, notice: "Successfully updated branch"
+			redirect_to @branch
+			flash[:notice] = "Successfully updated branch"
```

Comment:
I was reading some code another developer wrote when I was looking at how to get the flash messaging to work.  I thought it was formatted this way to make the code work.  I see now it can still be formatted the old way.